requirements/scripts.in

```diff
@@ -0,0 +1,3 @@
+atlassian-python-api      # provides an interface for creating Confluence pages
```

Comment:
We usually try to include a leading comment in each input requirements file describing what this group of dependencies is used for.